build/README.md

```diff
@@ -226,9 +226,9 @@ installing Agones next.
 
 To prepare building and pushing images, let's set the REGISTRY environment variable to point to our new project.
 You can [choose any registry region](https://cloud.google.com/container-registry/docs/pushing-and-pulling#choosing_a_registry_name)
```

Comment:
This link is for GCR (not artifact registry). If we are suggesting that folks use GAR instead of GCR are there other pre-requisites that need to be updated (like creating a registry in the project)?